pkg/galaxy/landscaper.go

```diff
@@ -155,7 +155,14 @@ func (l *Landscaper) loadKubeClient() error {
 // NewLandscaper instance a new Landscaper object.
 func NewLandscaper(
 	cfg *LandscaperConfig, kubeCfg *KubernetesConfig, env *Environment, ctxs []*Context,
-) *Landscaper {
```

Comment:
@eimarfandino formatting here seems a bit off, would you check?

Reply:
@otaviof i have pushed it